lib/qesapdeployment.pm

```diff
@@ -263,43 +278,61 @@ sub qesap_pip {
 =head3 qesap_pip_install
 
   Install all Python requirements of the qe-sap-deployment
-  in a dedicated virtual environment
+  in a dedicated virtual environment.
+  This function has no return code but it is expected to die
+  if something internally fails.
 =cut
 
 sub qesap_pip_install {
     my %paths = qesap_get_file_paths();
     my $pip_install_log = '/tmp/pip_install.txt';
     my $pip_ints_cmd = join(' ', qesap_pip(), 'install --no-color --no-cache-dir',
-        '-r', "$paths{deployment_dir}/requirements.txt",
-        '|& tee -a', $pip_install_log);
```

Comment:
the pipe to tee is no more delegated to each caller but added at once for all of them directly within qesap_venv_cmd_exec.